docs/pubsub-usage.rst

```diff
@@ -116,41 +127,42 @@ Create a new pull subscription for a topic:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> from gcloud.pubsub import Subscription
-   >>> topic = Topic('topic_name')
-   >>> subscription = Subscription('subscription_name', topic)
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
+   >>> subscription = pubsub.Subscription('subscription_name', topic)
```

Comment:
Wouldn't we recommend that you get a subscription from a topic object?

```python
client = pubsub.Client()
topic = client.topic('topic-name')
sub = topic.subscription('subscription-name')
```

?

Also seems weird that I get a topic from a client, but a subscription directly from the module, if anything, wouldn't it be:

```python
client = pubsub.Client()
topic = client.topic('name')
sub = client.subscription('name', topic=topic)
sub_alternate = topic.subscription('name')  # Or get_subscription?
```

(I'm not saying get rid of the `topic` argument in the subscription constructor, but it seems like we'd want to say "the recommended way" is to get a topic, then pull a subscription from the topic.)